src/ert/config/ert_config.py

```diff
@@ -120,7 +142,11 @@ def __post_init__(self) -> None:
         )
         self.enkf_obs: EnkfObs = self._create_observations(self.observation_config)
 
-        self.observations: Dict[str, xr.Dataset] = self.enkf_obs.datasets
+        self.observations: Dict[str, polars.DataFrame] = self.enkf_obs.datasets
+
+    def write_observations_to_folder(self, dest: Path) -> None:
```

Comment:
This is a nitpick, but should this function be here? Maybe it belongs with the observations?

Reply:
This relates to the `.to4` migration, it used to have hardcoded .to_netcdf() within it, and now it is moved to `ert_config`, so we can change the ert config obs writing logic without changing the migrations (also requiring special-casing in the `.to7` migration)

Reply:
Moved it to enkf_obs